### PR TITLE
Send users to northstar

### LINKED ISSUE #4276: User login events > northstar
Send user login events into Northstar, either via api.ds.org or directly into northstar

### PR DESCRIPTION
**Pull request by angaither**
When a user logs into dosomething, send that user into Northstar.
- Locally in vagrant to test you have to add `10.0.2.2 northstar.dev`  to `/etc/hosts`
  - there's a PR DoSomething/ansible-vagrant/pull/3 to do that
- Northstar module needs to be enabled

Fixes #4276

### CODE REVIEW

File: lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module [hunk 1]
```diff
@@ -28,14 +34,114 @@ function dosomething_northstar_menu() {
...
+      'X-DS-REST-API-Key' => NORTHSTAR_APP_KEY,
+      'Content-Type' => 'application/json',
+      'Accept' => 'application/json',
+      ),
+    );
+
+  return $client;
+}
+//@TODO: use this once we can use guzzle.
+// class Northstar {
```
Comment (blisteringherb) [inline lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module:114]: Did you want to include this commented out code or just add it after guzzle is avail?

**Comment by blisteringherb** [comment]
You mentioned the Northstar module needs to be enabled, should it be included in the makefile?

File: lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module (hunk 1 above)
Comment (angaither) [inline lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module:114]: Yeah, I mean might as well leave it in, as it the DREAM code we can use when guzzle is available.

**Comment by angaither** [comment]
@blisteringherb I don't want this enabled on stage, since we don't have a corresponding QA for northstar
so I left it out of the makefile

**Comment by blisteringherb** [comment]
:+1:

**Pull request merged**